src/sentry/conf/server.py

```diff
@@ -3196,6 +3196,7 @@ def custom_parameter_sort(parameter: dict) -> tuple[str, int]:
 MAX_MORE_SLOW_CONDITION_ISSUE_ALERTS = 300
 MAX_FAST_CONDITION_ISSUE_ALERTS = 500
 MAX_QUERY_SUBSCRIPTIONS_PER_ORG = 1000
+MAX_MORE_FAST_CONDITION_ISSUE_ALERTS = 1000
```

Comment:
🤨 hmmmm dang. wish we had chosen verbage better than 'max more fast' and 'max more slow' 😭

Reply:
hahah it's true.. if you look at the commit history on the pr i thought about changing it... 🤣